app/scripts/store/dimStoreItem.directive.js

```diff
@@ -240,14 +215,5 @@
     default:
       vm.draggable = true;
     }
-
-    settings.getSettings()
```

Comment:
really great work here removing this. lot less going on moving that out. calling getSettings() for EVERY item was pretty intense... even if most of it is cached, it's still being passed through promises.